app/src/scripts/dashboard/dashboard.datasets.jsx

```diff
@@ -108,13 +108,15 @@ class Datasets extends Reflux.Component {
             message="The dataset server failed to respond."
             className="loading-wrap fade-in">
             <PanelGroup>
-              {this.state.datasets.loading ? (
-                <Timeout timeout={20000}>
-                  <Spinner active={true} />
-                </Timeout>
-              ) : (
-                results
-              )}
+              <Panel>
```

Comment:
This leads to some extra padding around the datasets and jobs. The PanelGroup component here might be unnecessary (since it contained no Panel components anyways) but removing it likely requires some CSS updates.

![screenshot-2018-1-12 openneuro](https://user-images.githubusercontent.com/11369795/34889157-df4dcdc0-f781-11e7-976c-f2b6ff071338.png)

Reply:
great catch!